ci/build_windows.py

```diff
@@ -33,16 +33,43 @@
 import zipfile
 from distutils.dir_util import copy_tree
 from enum import Enum
-from subprocess import check_call
+from subprocess import check_call, call
+import urllib.request
 
 from util import *
 
+
+# Fix for broken PATH with newline inserted presumably by VS studio installation of SQL server or
+# other component which makes visual studio stop working.
+os.environ['PATH']=os.environ.get('PATH').replace('\n','')
+
 KNOWN_VCVARS = {
+    # https://gitlab.kitware.com/cmake/cmake/issues/18920
     'VS 2015': r'C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\x86_amd64\vcvarsx86_amd64.bat',
-    'VS 2017': r'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsx86_amd64.bat'
+    'VS 2017': r'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars64.bat',
+    'VS 2019': r'C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars64.bat'
 }
 
 
+def get_cuda_arch(default='"5.2"'):
+    # cuda arch auto detection in cmake doesn't work on windows ci
```

Comment:
The autodetection works fine on Windows. But we are compiling on a CPU instance. Can you remove this function and specify the MXNET_CUDA_ARCH argument as is done before? This function adds unnecessary complexity.

Reply:
Mistakenly added due to misconception, already removed. Sorry about this.

Reply:
No worries. Thanks for your work on this.